src/swagger-specs/query-service.yaml

```diff
@@ -36,6 +36,8 @@ tags:
     description: Schedules let you perform SQL functions from a start date to an end date.
   - name: "Query-Templates"
     description: Query templates let you create, store and execute any of the query as adhoc or schedule service.
+  - name: "Alert-Subscriptions"
+    description: Alert subscriptions allow you to receive notifications on the different statuses of both ad hoc and scheduled queries. Alerts can be received by email, within the Platform UI, or both. 
```

Comment:
Is there a reason why "Query-Templates" and "Alert-Templates" are hyphenated in their tag names, but not in the descriptions? The tag names are meant to describe the entity/resource that their endpoints manipulate in plain terms, rather than to emulate how the endpoint namespace is styled.

Swaggers also follow our capitalization rules re: titles, so: "Alert subscriptions"